src/ui/waveform.ts

```diff
@@ -3,14 +3,31 @@ import * as vscode from 'vscode';
 import { CXXRTLDebugger } from '../debugger';
 // @ts-ignore
 import embedHtml from '../surfer/embed.html';
+import { ILink, Packet } from '../cxxrtl/link';
+import { ClientPacket } from '../cxxrtl/proto';
+import { Variable } from '../model/variable';
+
+export class ClientPacketString {
+    constructor(public inner: string) { }
+}
+export class ServerPacketString {
+    constructor(public inner: string) { }
+}
```

Comment:
These classes don't seem like they're doing anything useful?

Reply:
I added them because I wanted type safety when passing around different kinds of strings